### ISSUE TITLE
Create goal percentage coverage analysis tool

### ISSUE THREAD

**Issue by adamantivm** [moveit/moveit_ros#69]
Create a simple tool / script to parse the benchmark output and produce percentage-of-goals-reachable in an easily consumable way

**Comment by isucan** [comment]
Would this be covered by #99 ?

**Comment by adamantivm** [comment]
We should all have a short sync meeting to discuss these things. It is becoming a bit hard to understand what each of us has in mind from the tickets, imho.

**Comment by isucan** [comment]
i agree with @adamantivm ; this would allow us to consolidate the tickets we have.

**Comment by marioprats** [comment]
Well, I think this ticket was assuming the current benchmark tool was
already providing the info we needed but in an unprocessed way. We
discovered that it does not provide the required info, so a new tool is
needed (#99). However, I think this ticket will still be needed, but
getting input from the new tool too. I agree we need a meeting ;)

**Comment by mateiciocarlie** [comment]
Yes, these are separate problems. Let me restate:

The goal of this ticket is to parse the results of benchmarking and provide human-readable output.

If the benchmark being run is the "traditional" planning benchmark, then the output it:
- % of states that are SOLVED

If the benchmark is the kinematics_benchmark tool from #99, then the output would be:
- % of states that are REACHABLE (in range, even if reaching them requires putting the arm in collision)
- % of states that COLLISION-FREE (REACHABLE in a collision-free fashion).

I actually did try to set up a sync meeting last week, but it turned out to be impossible to get a slot that worked for everybody. Just too many constraints.

I did close a ticket as a duplicate of this one though (#3 in the kinematics_analysis repo)

I will try to sync things up and consolidate tickets in our overall task ticket.
